core/parser/parser_test.go

```diff
@@ -2,58 +2,344 @@ package parser_test
 
 import (
 	"github.com/otrego/clamshell/core/parser"
+	"reflect"
 	"testing"
 )
 
-func TestCreate(t *testing.T) {
-	s := `
+var sgfs = map[string]struct {
+	raw string
+	gm  string
```

Comment:
Hmmm. I wonder if we could make this more generic

`type PropMap map[string][]string`

then you could have

`type Path string`

expectedProperties map[Path]PropMap

where a path looks like a treepath from glift: https://github.com/Kashomon/glift/blob/master/deps/glift-core/rules/treepath.js#L40

It's a very useful concept handling SGFs

Reply:
Maybe we could validate for just root right now and add treepath support later. Treepath, while very useful, is a rather complex concept.